app/views/users/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<!--<form accept-charset="UTF-8" action="/users" method="post">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="username">UserName:</label>
+  <input type="text" id="username" name="user[username]"><br><br>
```

Comment:
Your form label and input tag should follow this format:
```html
  <label for="user[username]">Username</label>
  <input name="user[username]" type="text" />
```
_Take note of the `for` attribute of the label and `name` attribute of the input tag._

Follow this style for other form elements in your **first** commented `<form>` tag